### DISCUSSION TITLE
Topolvm deployment failure on single node kubernetes cluster

### DISCUSSION THREAD

**Discussion by Madhan-SWE** [topolvm/topolvm#604] [Q&A] [answered]
Hi I am trying to deploy topolvm on my single node kubernetes cluster.
1. I set up the lvm and created a set up the service file and lvmd service is running
2. created necessary volume group
3. Installed the cert manager as followed `kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml`
4. Enabled `storageCapacityTracking` and disabled `scheduler` and `podMutatingWebhook` in the values file
5.added label `kubectl label namespace kube-system topolvm.io/webhook=ignore` and configured storage class in the values file
6.created namespace `topolvm-system` and ignore web hook by adding label `topolvm.io/webhook=ignore` to the namespace
7.Deploying the helm chart fails as below:

```
[root@madhank8s2 topolvm]# helm upgrade --namespace=topolvm-system -f values.yaml -i topolvm topolvm/topolvm
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /root/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /root/.kube/config
Release "topolvm" does not exist. Installing it now.
coalesce.go:220: warning: cannot overwrite table with non table for topolvm.controller.affinity (map[])
Error: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "topolvm-node" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "topolvm-scheduler" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"
ensure CRDs are installed first]
```

Any ideas about this issue?

**Comment by pluser** [comment, marked as answer]
Thank you for your question!

Which Kubernetes version are you using? TopoLVM does not currently support Kubernetes version 1.25. However, even if you are using that version, you can try disabling PodSecurityPolicy in [values.yaml](https://github.com/topolvm/topolvm/blob/aaff5ea8a07415fe9044d269e0858eac1cac18d3/charts/topolvm/values.yaml#L545) at your own risk if necessary.

**Reply by Madhan-SWE** [reply]
Sure, will try it on the previous version of kubernetes. thank you
